sdk-ext-lib/src/main/java/cash/z/ecc/sdk/ext/ZatoshiExt.kt

```diff
@@ -0,0 +1,16 @@
+@file:Suppress("ktlint:filename")
+
+package cash.z.ecc.sdk.ext
+
+import cash.z.ecc.android.sdk.ext.convertZatoshiToZec
+import cash.z.ecc.android.sdk.ext.convertZecToUsd
+import cash.z.ecc.android.sdk.ext.toUsdString
+import cash.z.ecc.android.sdk.model.Zatoshi
+import java.math.BigDecimal
+
+// TODO [#578] https://github.com/zcash/zcash-android-wallet-sdk/issues/578
+@Suppress("MagicNumber")
+fun Zatoshi.toUsdString() =
```

Comment:
As we're moving from basic scaffolding towards distributing internal builds of the app for testing, it would be great not to surface anything to the user that's broken/wrong.

I might suggest that we clean up this API now and have it return the "unavailable" state so that we can show that in the UI for now.

I am going to play with a few things here and add a few suggestions a bit later.

Reply:
Yes, good idea. I've refactored the function a bit.